app/_layout.tsx

```diff
@@ -26,6 +27,22 @@ import { useAppStore } from "~/lib/state/appStore";
 import { useColorScheme } from "~/lib/useColorScheme";
 import { registerForPushNotificationsAsync } from "~/services/Notifications";
 
+Sentry.init({
+  dsn: process.env.SENTRY_DSN,
+
+  // Adds more context data to events (IP address, cookies, user, etc.)
+  // For more information, visit: https://docs.sentry.io/platforms/react-native/data-management/data-collected/
+  sendDefaultPii: true,
+
+  // Configure Session Replay
+  replaysSessionSampleRate: 0.1,
```

Comment:
This is invasive isn't it? we need to respect user privacy and only catch actual errors?

Reply:
Sentry removes all the text fields (they've claimed this during setup and also confirmed by checking replay [here](https://getalby.sentry.io/issues/6770150912/?project=4507890509676544&query=is%3Aunresolved&referrer=issue-stream))

But yeah I'm also unsure if we want this, best to remove :+1: